src/Razor/test/testapps/SimpleMvc/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -0,0 +1,115 @@
+<!--
```

Comment:
I grabbed the design time targets that were on my machine. I'll add a work item to clear it up as part of removing the workaround. If you feel strongly against adding these, I can remove these tests.

Reply:
Ya i'd nuke the test given that we're comparing across previews. This is something better caught during validation.